docs/content/en/docs/install/integrate.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Integrate KLT
+description: How to integrate the Keptn Lifecycle Toolkit into your Kubernetes cluster
+icon: concepts
+layout: quickstart
+weight: 45
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Use Kubernetes annotations and labels
+to integrate the Keptn Lifecycle Toolkit into your Kubernetes cluster.
```

Comment:
High level integration instructions:

1. Annotate your workload(s)
2. Create a `KeptnApp` custom resource which references those workloads
3. Create any `KeptnTask`s you need
4. Enable the target namespace by annotating it.

This flow is demonstrated here: https://www.youtube.com/watch?v=Hh01bBwZ_qM&t=6s

KLT will now be working on those workloads.

Reply:
I added the four-step summary to the file.  We obviously need more details here but let's get this much merged first.